pkgs/html/lib/dom_parsing.dart

```diff
@@ -43,7 +43,14 @@ class TreeVisitor {
   void visitText(Text node) => visitNodeFallback(node);
 
   // TODO(jmesserly): visit attributes.
-  void visitElement(Element node) => visitNodeFallback(node);
+  void visitElement(Element node) {
+    if (node.localName == 'br') {
```

Comment:
This can be separate `isElementBr` function. Is this if statement enough to recognize it?